.github/workflows/deploy.yml

```diff
@@ -146,7 +146,7 @@ jobs:
           path: web/public/
 
   publish:
-    if: github.repository_owner == 'theforeman' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/[0-9]+.[0-9]+'))
+    if: github.repository_owner == 'theforeman' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/2.') || startsWith(github.ref, 'refs/heads/3.') || startsWith(github.ref, 'refs/heads/4.'))
```

Comment:
I've said it before, but I think we should follow Foreman's convention. That means `develop` for nightly and `x.y-stable` for stable branches. Then you can check it with an `endsWith` instead of a number.

Reply:
Well it looks like the glob syntax does not work with startsWith despite some advice on stackoverflow.com.